### PR TITLE
New responsive_design@es.md and ...

### PR DESCRIPTION
**Pull request by SidVal**
New translate for responsive_design.md :es: 
and some changes to `messages.po`

### CODE REVIEW

File: content/docs/guides/responsive_amp/responsive_design@es.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+## Introducción
+
+El diseño web responsivo consiste en crear páginas web fluidas que responden a las necesidades de su usuario, páginas que se ajustan al tamaño de la pantalla y la orientación de su dispositivo. Usted puede lograr esto fácilmente en AMP. AMP admite todas las categorías de pantallas y dispositivos y además proporciona componentes responsivos integrados.
+
+En esta guía, le mostraremos cómo puede implementar fácilmente estos fundamentos de responsivos en AMP:
+
+- [Controlando el viewport](#controlando-el-viewport)
+- [Creando el layout responsivo](#creando-el-layout-responsivo)
+- [Medios de escala](#medios-de-escala-para-la-pagina)
```
Comment (bpaduch) [inline content/docs/guides/responsive_amp/responsive_design@es.md]: Page anchor should be: `#medios-de-escala-para-la-página`

File: content/docs/guides/responsive_amp/responsive_design@es.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+```html
+<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+```
+
+Éstas son las configuraciones típicas del viewport que utilizarías para un sitio responsivo. Aunque `initial-scale = 1` no es necesario para una página AMP válida, se recomienda porque establece el nivel de zoom en 1 cuando se cargue la página por primera vez.
+
+
+## Creando el layout responsivo
+
+En el diseño responsivo, puede utilizar CSS [`@media`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media) para adaptar el estilo de su página web para la variedad de la pantalla dimensiones sin tener que alterar el contenido de la página. En AMP, puede seguir utilizando las mismas consultas CSS `@media`. Además, para un control más preciso sobre un elemento AMP, se puede especificar el atributo `media` en el elemento. Esto es particularmente útil cuando se necesita para mostrar u ocultar un elemento en base a una consulta de medios. Ver la sección [Cambiando la dirección de arte de una imagen](#cambiando-la-direccion-artistica-de-una-imagen) para un ejemplo que utiliza el atributo `media`.
```
Comment (bpaduch) [inline content/docs/guides/responsive_amp/responsive_design@es.md]: Page anchor should be: `#cambiando-la-dirección-artística-de-una-imagen`

File: content/docs/guides/responsive_amp/responsive_design@es.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+<div>
+<amp-iframe height="174" layout="fixed-height" sandbox="allow-scripts allow-forms allow-same-origin" resizable src="https://ampproject-b5f4c.firebaseapp.com/examples/responsive.youtube.embed.html"> <div overflow tabindex="0" role="button" aria-label="Show more">Show full code</div> <div placeholder></div> </amp-iframe></div>
+
+Hay más tipos de videos que tú puedes agregar a tus páginas AMP. Para más detalles, mira la lista de [media components](https://www.ampproject.org/docs/reference/components#media) -en inglés.
+
+### Mostrando imágenes responsivas
+
+Las imágenes forman una gran parte de una página web (aproximadamente el 65% de los bytes de la página). Como mínimo, las imágenes deben ser visibles en varios tamaños de pantalla y orientaciones (es decir, el usuario no tiene que desplazarse, pellizcar o ampliar para ver toda la imagen). Esto se hace fácilmente en AMP a través del atributo `"layout=responsive"` (ver [Incluir imágenes en AMP](https://www.ampproject.org/docs/guides/amp_replacements)). Además de la imagen básica responsiva, es posible que desee publicar varios recursos de imágenes para:
```
Comment (bpaduch) [inline content/docs/guides/responsive_amp/responsive_design@es.md]: Missing link to stats on 65% - should go to http://httparchive.org/interesting.php#bytesperpage

File: content/docs/guides/responsive_amp/responsive_design@es.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+- [Sirve imágenes nítidas para obtener la resolución correcta](#mostrando-imagenes-nitidas-para-la-resolucion-correcta)
```
Comment (bpaduch) [inline content/docs/guides/responsive_amp/responsive_design@es.md]: Anchors need to be corrected:
- `#mostrando-imágenes-nítidas-para-la-resolución-correcta`
-  `#cambiando-la-dirección-artística-de-una-imagen`
- `proporcionar-imágenes-optimizadas`

File: content/docs/guides/responsive_amp/responsive_design@es.md (hunk 1 above)
Comment (SidVal) [inline content/docs/guides/responsive_amp/responsive_design@es.md]: Ah, OK... I did not know that URL accepts accents :+1:

**Pull request merged**